### ISSUE TITLE
Maploom installed, but not loading

### ISSUE THREAD

**Issue by gxcodes** [ROGUE-JCTD/django-maploom#5]
Hi, I've installed maploom with 'pip install' and it return 'Successfully installed django-maploom' despite two warnings about no files found matching 'CHANGES' and '*.ttf'.

I followed the 'Project Configuration' and 'Adding in Geonode' steps, but after I restart apache2, maploom is still not working. Inspecting the new maps page, the iframe for maploom seems to be working but does not load anything.

Geonode is working perfectly using apt-get install method on ubuntu server 14.04.4. Any ideas to make maploom work?

**Comment by garnertb** [comment]
Try a `python manage.py collectstatic` from your geonode root.

**Comment by gxcodes** [comment]
I've copied the static files manually, but also use the collectstatic method per your suggestion. Both cases worked, I can now have the maploom loaded in the maps view.

But this led me to another question, everytime I load maploom, it started with a modal error containing only "script_error" message. After clicking that, all other functionalities seem to work though, despite all the buttons showing their placeholder title instead of the text intended, for example "legend_title" and "map_layers".

Any suggestion to solve that?

**Comment by garnertb** [comment]
You'll need to provide some more information in order for me to help.  Can you open up developer tools and include any exceptions in this issue?

The fact that you are seeing the raw strings means that something is failing before those string keys get replaced with the appropriate locally translated string.

**Comment by garnertb** [comment]
Closing.  Feel free to reopen if this is still an issue and you can provide more information.

**Comment by gxcodes** [comment]
Hi, really sorry for the delay, it's holiday season over here.

I haven't been able to sort it out. On the console, I notice the following error:

> Error: server is null
> this.addLayer@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:165801:1
> this.loadLayers/addLayersForServer/<@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:166277:15
> goog.array.forEach<@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:34206:7
> this.loadLayers/addLayersForServer@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:166269:11
> this.loadLayers/</<@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:166298:15
> qFactory/defer/deferred.promise.then/wrappedErrback@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11561:31
> qFactory/createInternalRejectedPromise/<.then/<@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11694:29
> $RootScopeProvider/this.$get</Scope.prototype.$eval@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:12664:16
> $RootScopeProvider/this.$get</Scope.prototype.$digest@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:12476:15
> $RootScopeProvider/this.$get</Scope.prototype.$apply@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:12768:13
> bootstrap/doBootstrap/<@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:1470:9
> invoke@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:3972:14
> bootstrap/doBootstrap@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:1468:5
> bootstrap@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:1482:12
> angularInit@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:1395:5
> @http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:21910:5
> x.Callbacks/l@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11:24877
> x.Callbacks/c.fireWith@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11:25702
> .ready@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11:2898
> S@http://192.168.100.9/static/maploom/assets/MapLoom-1.5.0.js:11:551
>  MapLoom-1.5.0.js:10055:18
> 
> ==== onErrorCallback, error msg: TypeError: server is null

Here's some screenshot:
<img width="1025" alt="screen shot 2016-07-07 at 10 51 28" src="https://cloud.githubusercontent.com/assets/5865879/16642098/9393abc0-4431-11e6-947c-a03c4c5ea830.png">

**Comment by gxcodes** [comment]
Just a little update: after I changed the geonode default language back to English, all the buttons and text are displayed correctly, so probably that problem is caused by lack of translation strings. 

But the script_error remains, there's still error modal everytime the map view is opened for the first time. The error script now says: 

> The following script error has occurred: "TypeError: server is null". It is recommended that you reload the web page to resolve any potential complications. 

Any idea to overcome this? Thanks!

**Comment by gxcodes** [comment]
Managed to find it out myself. Yet, no real solution, just a workaround to make it 'work'.
1. Raw text problem is caused by the lack of translation string. The solution is either to provide the translation, or use english on MapLoom pages.
2. The "TypeError: server is null" is caused by the base layer I used (bing/google, forgot which one, maybe both). Disable them for now until I could figure itu out, I stick to OSM.
